packages/webdriverio/src/utils/interception/webdriver.ts

```diff
@@ -8,6 +8,8 @@ import { ERROR_REASON } from '../../constants'
  * compliant backend.
  */
 export default class WebDriverInterception extends Interception {
+    mockId: number = NaN;
```

Comment:
Should this be initialized somehow here or then should we check that `mockId !== undefined` before calling the methods that require mockId?

Reply:
it should be defined as optional

Reply:
Then how do you suggest dealing with those?
![image](https://user-images.githubusercontent.com/15076656/96556950-4bd2e580-12c2-11eb-97df-b274f5d9615e.png)

Reply:
I could cast, maybe